### PR TITLE
Use HTML "title" attribute on plugin's button

### PR DESCRIPTION
**Pull request by hermancaldara**
Updates the generator to create the package with buttons with attr "title".

### CODE REVIEW

File: generators/app/templates/src/constants.js [hunk 1]
```diff
@@ -5,5 +5,6 @@
  */
 
 export default {
+  PLUGIN_NAME: "<%= inputs.pluginName %>"
```
Comment (marcelometal) [inline generators/app/templates/src/constants.js]: Hi @hermancaldara, please add a comma here ;)

**Review by marcelometal — CHANGES_REQUESTED** [review]
Hi @hermancaldara, Could you please fix the syntax error?

Thanks for contributing to Megadraft!

**Comment by hermancaldara** [comment]
Done! Sorry for that, @marcelometal.

**Comment by marcelometal** [comment]
Thanks for contributing to Megadraft!

**Pull request merged**